openc3-cosmos-init/plugins/packages/openc3-tool-common/src/components/widgets/GraphWidget.js

```diff
@@ -114,5 +114,21 @@ export default {
           break
       }
     })
+    if (this.screen) {
+      this.items.map((item) =>
+        this.screen.addItem(
+          `${item.targetName}__${item.packetName}__${item.itemName}__${item.valueType}`,
+        ),
+      )
+    }
+  },
+  destroyed() {
+    if (this.screen) {
+      this.items.map((item) =>
+        this.screen.deleteItem(
+          `${item.targetName}__${item.packetName}__${item.itemName}__${item.valueType}`,
+        ),
+      )
+    }
```

Comment:
This code adds the graph items to the screen which will get called by `get_tlm_values`. It's unnecessary for graphs but the only way that the screen can validate the items exist. Otherwise the graph just starts graphing nothing ... the time scale moves but nothing shows up. It's a little inefficient but since we have screen caching for items I felt it was better to avoid the weird graph behavior when items don't exist.